server/index.js

```diff
@@ -12,6 +12,8 @@ const app = express();
 app.get("*/dist/:fileName", staticFilesController.getFile);
 app.get(/^((?<!\/dist\/).)*/, indexController.getPage);
 
-app.listen(3000, function () {
-	console.log("express server started on localhost:3000");
+app.listen(process.env.PORT || 3000, function () {
```

Comment:
Чаще всего в места где нужна анонимная функция, используют стрелочные функции:
```suggestion
app.listen(process.env.PORT || 3000, () => {
```